gui/pages/kernel_options.go

```diff
@@ -0,0 +1,292 @@
+
+	"github.com/clearlinux/clr-installer/kernel"
+	"github.com/clearlinux/clr-installer/log"
+	"github.com/clearlinux/clr-installer/model"
+	"github.com/clearlinux/clr-installer/utils"
+	"github.com/gotk3/gotk3/gtk"
+	"strings"
+)
+
+// KConfigPage is a page to change kernel installation configuration
+type ConfigKernelPage struct {
+	controller Controller
+	model      *model.SystemInstall
+	box        *gtk.Box
+	addEntry   *gtk.Entry
+	remEntry   *gtk.Entry
+	addLabel   *gtk.Label
+	remLabel   *gtk.Label
+	scroll     *gtk.ScrolledWindow
+	list       *gtk.ListBox
+	data       []*kernel.Kernel
+	selected   *kernel.Kernel
+}
+
+// NewConfigKernelPage returns a new NewConfigKernelPage
+func NewConfigKernelPage(controller Controller, model *model.SystemInstall) (Page, error) {
+	kernelArgsHelp :=
+		utils.Locale.Get("Note: The boot manager tool will first include the \"Add Extra Arguments\" items.")
+	kernelArgsHelp = kernelArgsHelp + "\n" +
+		utils.Locale.Get("Then, the \"Remove Arguments\" items are removed.")
+	kernelArgsHelp = kernelArgsHelp + "\n" +
+		utils.Locale.Get("The final argument list contains the kernel bundle's configured arguments and the ones configured by the user.")
+
+	data, err := kernel.LoadKernelList()
+	if err != nil {
+		return nil, err
+	}
+	page := &ConfigKernelPage{
+		controller: controller,
+		model:      model,
+		data:       data,
+	}
+
+	// Box
+	page.box, err = setBox(gtk.ORIENTATION_VERTICAL, 0, "box-page")
+	if err != nil {
+		return nil, err
+	}
+
+	// ScrolledWindow
+	page.scroll, err = setScrolledWindow(gtk.POLICY_NEVER, gtk.POLICY_AUTOMATIC, "scroller")
+	if err != nil {
+		return nil, err
+	}
+	page.scroll.SetMarginStart(common.StartEndMargin)
+	page.scroll.SetMarginEnd(common.StartEndMargin)
+	page.box.PackStart(page.scroll, true, true, 0)
+
+	// ListBox
+	page.list, err = setListBox(gtk.SELECTION_SINGLE, true, "list-scroller")
+	if err != nil {
+		return nil, err
+	}
+	if _, err := page.list.Connect("row-activated", page.onRowActivated); err != nil {
+		return nil, err
+	}
+	page.scroll.Add(page.list)
+
+	// Create list data
+	for _, v := range page.data {
+		name := v.Name
+		desc := v.Desc
+
+		//Add default label to kernel already in model
+		if page.model.Kernel != nil {
+			if v.Bundle == page.model.Kernel.Bundle {
+				name = v.Name + utils.Locale.Get(" (default)")
+			}
+		}
+
+		box, err := setBox(gtk.ORIENTATION_VERTICAL, 0, "box-list-label")
+		if err != nil {
+			return nil, err
+		}
+
+		labelName, err := setLabel(name, "list-label-name", 0.0)
+		if err != nil {
+			return nil, err
+		}
+		box.PackStart(labelName, false, false, 0)
+
+		labelDesc, err := setLabel(desc, "list-label-desc", 0.0)
+		if err != nil {
+			return nil, err
+		}
+		box.PackStart(labelDesc, true, true, 0)
+
+		page.list.Add(box)
+	}
+
+	// addLabel: Tell users they can add kernel command lines below
+	addText := utils.Locale.Get("Add Extra Arguments:")
+
+	page.addLabel, err = setLabel(addText, "label-addLabel", 0.0)
```

Comment:
Please change the style to the "label-entry" which we are using for the add user screen for consistency.

Reply:
Done.